### ISSUE TITLE
Commands?

### ISSUE THREAD

**Issue by damncabbage** [paf31/purescript-yargs#7]
I've been trying to come up with a way to add `.command()` support to this library. I'm drawing a complete blank. Any ideas? :cold_sweat:

**Comment by paf31** [comment]
Perhaps the best way is to use the version which takes a function?

**Comment by damncabbage** [comment]
Yeah. I started there, but I struggled to plumb it all together into something that'll let you match some commands up to a some-type of options, or... I'm struggling to think of the interface to all this.

(I had a brainstorm with @anttih [a couple of months back](http://ircbrowse.net/browse/purescript?id=243556&timestamp=1452525101#t1452525101), but came up blank.)
